lib/Sema/ConstraintSystem.h

```diff
@@ -407,8 +407,10 @@ enum ScoreKind {
   SK_ScalarPointerConversion,
   /// A conversion from an array to a pointer of matching element type.
   SK_ArrayPointerConversion,
+  /// A conversion to an empty existential type ('Any' or '{}').
```

Comment:
Okay, but that's never been a valid spelling for an empty protocol composition. It's not important enough to reset your smoke test for, but if you do another round here can you remove it?